### PR TITLE
add fastlane for F-Droid

### PR DESCRIPTION
**Pull request by linsui**
F-Droid will fetch descriptions from here.

### CODE REVIEW

**Comment by licaon-kter** [comment]
Can you steal some pics from https://play.google.com/store/apps/details?id=com.zulipmobile too? :)

**Comment by linsui** [comment]
If the zulip guys don't object, I can.

**Comment by gnprice** [comment]
For a screenshot of the app, here's a good one to use:
https://zulip.com/static/images/app-screenshots/zulip-android.png
https://github.com/zulip/zulip/blob/main/static/images/app-screenshots/zulip-android.png
It's already covered by the free license that applies to the Zulip software.

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The official Zulip mobile app.
```
Comment (gnprice) [inline fastlane/metadata/android/en-US/full_description.txt:1]: I'd prefer that the Zulip app in F-Droid not be described as "official".

In particular, I see at https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11516/diffs that there are a number of changes being made to the software, including removing some functionality. That's fine -- it's free software -- but it doesn't seem accurate to call the result "[t]he official Zulip" anything.

**Review by gnprice — COMMENTED** [review]
One comment on the descriptions, below.

**Comment by linsui** [comment]
I removed the official word and added the screenshot.

**Comment by gnprice** [comment]
Thanks for the revision! Looks good -- merging.

**Comment by gnprice** [comment]
BTW, before distributing a version on F-Droid, I'd urge you to update to use the newest release, v27.190, which we put out this week. It fixes a security vulnerability in previous versions.

(I'm guessing/hoping F-Droid has automation to keep up with updates in general, but since we're discussing here now I figured I'd mention it.)

**Comment by linsui** [comment]
> BTW, before distributing a version on F-Droid, I'd urge you to update to use the newest release, v27.190, which we put out this week. It fixes a security vulnerability in previous versions.

Of course!

> (I'm guessing/hoping F-Droid has automation to keep up with updates in general, but since we're discussing here now I figured I'd mention it.)

Yes, the update is automatically.

**Comment by linsui** [comment]
The version is not bumped in master and the 27.190 tag is out of the tree. I'll build from master so the version is still 289 but other changes are included anyway.

**Comment by gnprice** [comment]
Sounds good! Happy to see this happening.

**Pull request merged**